manual/cops_rspec.md

```diff
@@ -977,7 +977,7 @@ IgnoredWords | `[]` | Array
 
 Enabled by default | Supports autocorrection
 --- | ---
-Enabled | No
+Enabled | Yes
```

Comment:
I'm not sure how exactly this is generated, shouldn't we add `AutoCorrect: true` to `config/default.yml`?

Reply:
Nope.
AutoCorrect: true / AutoCorrect: false in config/default.yml just tells whether this autocorrect is enabled or disabled by default. It is a leftover from the times before we had a `SafeAutocorrect` option. 

This just tells whether the cop implements `autocorrect` at all.